conans/test/functional/command/upload_complete_test.py

```diff
@@ -253,6 +266,61 @@ def upload_error_with_config_test(self):
         client.run("upload Hello* --confirm --all")
         self.assertEqual(str(client.out).count("ERROR: Pair file, error!"), 6)
 
+    def upload_parallel_error_test(self):
+        """Cause an error in the parallel transfer and see some message"""
+        client = TestClient(requester_class=FailOnReferencesUploader, default_server_user=True)
+        client.save({"conanfile.py": GenConanfile()})
+        client.run('user -p password -r default user')
+        for index in range(4):
+            client.run('create . lib{}/1.0@user/channel'.format(index))
+        client.run('upload lib* --parallel -c --all -r default', assert_error=True)
+        self.assertIn("Connection fails with lib2 and lib4 references!", client.out)
+        self.assertIn("Execute upload again to retry upload the failed files", client.out)
+
+    def upload_parallel_success_test(self):
+        """Upload 2 packages in parallel with success"""
+
+        # For each file will fail the first time and will success in the second one
+        server = TestServer(users={"user": "password"}, write_permissions=[("*/*@*/*", "*")])
```

Comment:
I think the simplification of ``default_server_user=True`` can be used in other tests too.